.github/workflows/build.yml

```diff
@@ -40,3 +40,9 @@ jobs:
           context: .
           push: true
           tags: ${{ steps.login-ecr-vaec.outputs.registry }}/notification_api:${{ inputs.ref }}_${{ inputs.environment }}
+
+      - name: Dispatch Twistlock Workflow
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          gh workflow run -r 974-twistlock-cronjob -F image-tag=${{ inputs.ref }}_${{ inputs.environment }} twistlock.yml
```

Comment:
This should be master?

and `inputs.ref` should use the other style.

It seems to be working, but I'm surprised the ref and environment don't need quotes e.g.
`image-tag="${{ inputs.ref }}_${{ inputs.environment }}"`

Reply:
great catch!

Reply:
The output is correct in the action output, so I would like to keep it as is (the gh cli doesn't like quotes on the -F inputs, IIRC)